public/modules/mcq.css

```
@@ -239,7 +240,16 @@
 }
 
 .mcq-option-label {
-  display: none; /* Option labels (A, B, C) are not shown in the design */
+  /* Visually hidden; still in the wrapping label's accessible name. */
+  position: absolute;
+  width: 1px;
+  height: 1px;
+  padding: 0;
+  margin: -1px;
+  overflow: hidden;
+  clip: rect(0, 0, 0, 0);
```

Stylelint (17.14.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)